web/src/repogroups/RepogroupPage.tsx

```diff
@@ -105,8 +105,8 @@ export const RepogroupPage: React.FunctionComponent<RepogroupPageProps> = (props
                     interactiveModeHomepageMode={true}
                 />
             </div>
-            <div className="repogroup-page__content">
-                <div className="repogroup-page__column">
+            <div className="repogroup-page__content row">
```

Comment:
The class doesn't have any styles anymore
```suggestion
            <div className="row">
```